helm-charts/FATE/values-template-example.yaml

```diff
@@ -26,6 +26,12 @@ modules:
 
 backend: eggroll
 
+# host:
+  # fateboard: 9999.fateboard.abc.net
```

Comment:
the party name should not be part of mutable host.

Reply:
I think it's easier to understand